doc/source/install.rst

```diff
@@ -16,4 +16,26 @@ To install the latest development version, do
 
 NOTE: This version is considered UNSTABLE. DON'T use it in the production environment!
 
-Stable version will be released soon.
\ No newline at end of file
+Stable version will be released soon.
+
+Since we are using `plpython3` in Greenplum/Postgres to better use GreenplumPython we also need
+to install the package in the server.
+
+If you are using Greenplum you may need to install the following package on the server(plpython3 side)
```

Comment:
postgres also better to have dill, so, rephrase to:

[dill](https://github.com/uqfoundation/dill) as an optional dependency for GreenplumPython `plpython` side, provides convenient features like auto-importing modules in the `plpython` functions. Refer to [GPDB plpython document](https://docs.vmware.com/en/VMware-Tanzu-Greenplum/6/greenplum-database/GUID-analytics-pl_python.html#pip39) about how to install [dill](https://github.com/uqfoundation/dill) for Greenplum.



The example here won't be quite helpful I think, let's just point users to the gpdb python module doc, if it is not enough, let's fix the doc there.